examples/hostswithgroup.cf

```diff
@@ -0,0 +1,20 @@
+body common control
+{
+      bundlesequence => { "example" };
+      inputs => { "$(sys.libdir)/stdlib.cf" };
+}
+
+
+bundle agent example
+{
+  vars:
+
+    am_policy_hub::
+      "host_list" slist => hostswithgroup( "Linux", "name" );
```

Comment:
This example can also be a test right? You could maybe check if the host name `$(sys.host)` is in the `slist` and print something based on a condition using the `reports` promise and check the output?

Reply:
Examples as tests are great, but this an enterprise hub only function so I am not sure it works well for running the tests.